### ISSUE TITLE
Remove bootstrap dependency

### ISSUE THREAD

**Issue by jamiechong** [Owumaro/vue-date-range-picker#1]
It would be great if this component implemented its own CSS styles and didn't depend on Bootstrap.

**Comment by Owumaro** [comment]
I agree on this, however I would like to keep this "bootstrap-compatible".

For example, if someone is using bootstrap with custom variables, the plugin should use its rules.

I thought about copying/pasting bootstrap rules that I use in this plugin, however this would override custom bootstrap from the app.
I could use scoped css to avoid this, but by doing, custom bootstrap css in the app won't apply to the component.

I don't know if there is a best practice for this issue, if you think of something let me know :)
